.chef/knife.rb

```diff
@@ -4,4 +4,6 @@
 client_key               '/home/monika/tutorial/.chef/monika.pem'
 validation_client_name   'chef-validator'
 validation_key           '/home/monika/tutorial/.chef/chef-validator.pem'
-chef_server_url          'https://10.21.1.10:443'
+chef_server_url          'https://10.21.1.9:443'
+syntax_check_cache_path  '/home/monika/tutorial/.chef/syntax_check_cache'
+cookbook_path            '/home/monika/tutorial/cookbooks'
```

Comment:
Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.